RideShare/package.json

```diff
@@ -51,11 +45,11 @@
     "karma-coverage-istanbul-reporter": "~2.0.1",
     "karma-jasmine": "~2.0.1",
     "karma-jasmine-html-reporter": "^1.4.0",
-    "prettier": "^1.19.1",
+    "karma-junit-reporter": "^2.0.1",
+    "karma-sonarqube-unit-reporter": "0.0.21",
     "protractor": "~5.4.0",
     "ts-node": "~7.0.0",
     "tslint": "~5.15.0",
-    "tslint-config-prettier": "^1.18.0",
-    "typescript": "~3.5.3"
+    "typescript": "^3.4.5"
```

Comment:
Sonarqube needs "^" as opposed to "~" to recognize the Typescript version. Please include "^"